src/jfrMetadata.cpp

```diff
@@ -256,6 +256,32 @@ JfrMetadata::JfrMetadata() : Element("root") {
                 // when encountering a T_BYTE/F_ARRAY.
                 << field("data", T_STRING, "User Data"))
 
+            << (type("profiler.ProcessSample", T_PROCESS_SAMPLE, "Process Stats Sample")
+                << category("Operating System", "Process")
+                << field("startTime", T_LONG, "Start Time", F_TIME_TICKS)
+                << field("eventThread", T_THREAD, "Event Thread", F_CPOOL)
+                << field("pid", T_INT, "Process ID")
+                << field("ppid", T_INT, "Parent Process ID")
+                << field("name", T_STRING, "Process Name")
+                << field("cmdLine", T_STRING, "Command Line")
+                << field("uid", T_INT, "User ID")
+                << field("state", T_BYTE, "Process State")
+                << field("processStartTime", T_LONG, "Process Start Time")
+                << field("cpuUser", T_LONG, "User CPU Time")
+                << field("cpuSystem", T_LONG, "System CPU Time")
+                << field("cpuPercent", T_FLOAT, "CPU Percentage")
+                << field("normalizeCpuPercent", T_FLOAT, "Normalized CPU Percentage")
+                << field("threads", T_INT, "Thread Count")
+                << field("memSize", T_LONG, "Virtual Memory Size")
+                << field("memResident", T_LONG, "Resident Memory")
+                << field("memShared", T_LONG, "Shared Memory")
+                << field("memText", T_LONG, "Text Memory")
+                << field("memData", T_LONG, "Data Memory")
```

Comment:
I never saw these two useful.
In contrast, it's quite important to distinguish anonymous memory from file-mapped memory, but I don't see it here.